src/hotspot/share/gc/shared/stringdedup/stringDedupProcessor.cpp

```diff
@@ -64,6 +67,10 @@ StringDedup::Processor::Processor() : _thread(nullptr) {}
 
 void StringDedup::Processor::initialize() {
   _processor = new Processor();
+  if (UsePerfData && os::is_thread_cpu_time_supported()) {
```

Comment:
The if and EXCEPTION_MARK could be removed, because `create_counter()` does that internally.